### ISSUE TITLE
checker reports fialure on xs rule groups

### ISSUE THREAD

**Issue by mardaravicius** [Divy1211/xs-check#8]
checker reports failure on xs rule groups

```c
rule xsTestRule active group someGroup highFrequency priority 21 {
}
```

```
Error: ParsingError
   ╭─[.\default0.xs:9:30]
   │
 9 │ rule xsTestRule active group someGroup highFrequency priority 21 {
   │                              ────┬────
   │                                  ╰────── found someGroup expected something else
───╯
```

groups names are words after the group modifiers in xs rule and are valid xs.

**Comment by Divy1211** [comment]
Oh, it looks like group names without quotes don't work, thanks for the report, I can fix this

**Comment by Divy1211** [comment]
Fixed in v0.1.5
